### PR TITLE
Bidding UI

### PR DESCRIPTION
**Pull request by NoelleinMN**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,43 @@
...
+# Marketing-Melons-grp-project-
+
+Market place is where we have information about melons ,bidding process involved.
+
+Clone the project
+
+git clone https://github.com/deeptidhar/Marketing-Melons-grp-project-.git
+Go to the project directory
+
+cd forever-project
```
Comment (poemusica) [inline README.md:10]: that seems like the wrong directory name...

**Review by poemusica — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (poemusica) [reply README.md:10]: Just making a note in case you want to update it next time.

File: README.md [hunk 2]
```diff
@@ -1 +1,43 @@
...
+virtualenv env
+source env/bin/activate
+
+Install requirements
+ pip3 install requirements.txt
```
Comment (poemusica) [inline README.md:15]: `pip3 install -r requirements.txt`

Comment (poemusica) [reply README.md:15]: In the future, you can also add running the seed file (which creates the database, tables, and example data) and then running the server.

**Review by Quanisha704 — APPROVED** [review]
Great job today ladies. I am excited to run the app and see the outcome!

**Pull request merged**